_posts/2022-10-11-quarkus-newsletter-25.adoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: 'Quarkus Newsletter #25 - October'
+date: 2022-10-11
+tags: newsletter
+synopsis: The October edition of the Quarkus Newsletter; curated online stories, articles, and videos about Quarkus.
+author: jcobb
+bibquery: ["pub.urldate > '2022-09-13'", "pub.urldate <= '2022-10-12'"]
+---
+ 
+Its that time of the month... time for the latest Quarkus newsletter! Piotr Minkowski discusses how the Quarkus framework handles Java Virtual threads in "Quarkus ith Java Virtual Threads". "Tutorial: Build a Quarkus web app with Azure App Service on Linux and PostgreSQL" walks you through the process of building, configuring, deploying, and scaling Java web apps on Azure. Vamil Kumar's article; "Service discovery using SmallRye Stork with Quarkus" microservice based architecture where multiple services are interacting with each other to carry out a transaction, service discovery becomes an integral part as it provides a mechanism to register and find the services and how SmallRye Stork provides both features. Learn how to easily deploy and run serverless functions from the open source project Knative Quarkus Bench on IBM Cloud Code Engine in Scott Trent's "Using Knative Quarkus Bench to Perform Serverless Experiments on IBM Cloud Code Engine". Lastly, check out Pierre Guimon's great "Quarkus fundamentals" which gives you an overview of Quarkus framework, as well as explaining its interactions with GraalVM.
```

Comment:
```suggestion
It's that time of the month... time for the latest Quarkus newsletter! Piotr Minkowski discusses how the Quarkus framework handles Java Virtual threads in "Quarkus with Java Virtual Threads". "Tutorial: Build a Quarkus web app with Azure App Service on Linux and PostgreSQL" walks you through the process of building, configuring, deploying, and scaling Java web apps on Azure. Vamil Kumar's article; "Service discovery using SmallRye Stork with Quarkus" microservice based architecture where multiple services are interacting with each other to carry out a transaction, service discovery becomes an integral part as it provides a mechanism to register and find the services and how SmallRye Stork provides both features. Learn how to easily deploy and run serverless functions from the open source project Knative Quarkus Bench on IBM Cloud Code Engine in Scott Trent's "Using Knative Quarkus Bench to Perform Serverless Experiments on IBM Cloud Code Engine". Lastly, check out Pierre Guimon's great "Quarkus fundamentals" which gives you an overview of Quarkus framework, as well as explaining its interactions with GraalVM.
```

Also I'm unable to parse the following sentence: `Vamil Kumar's article; "Service discovery using SmallRye Stork with Quarkus" microservice based architecture where multiple services are interacting with each other to carry out a transaction, service discovery becomes an integral part as it provides a mechanism to register and find the services and how SmallRye Stork provides both features.`.

It would be great if you followed the one sentence per line rule as GitHub suggestions don't highlight the diff for long lines.